modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -436,7 +436,8 @@ def transform_service_branch
 
     def transformed_reserves_national_guard_service
       unit_phone = form_data&.dig('serviceInformation', 'reservesNationalGuardService', 'unitPhone')
-      unit_phone['phoneNumber'].delete!('-') if unit_phone.present? && unit_phone['phoneNumber'].include?('-')
+      unit_phone['phoneNumber'].gsub!(/[-\s]/, '') if unit_phone.present? && unit_phone['phoneNumber'].include?('-')
+      unit_phone['areaCode'].gsub!(/\s/, '') if unit_phone.present? && unit_phone['areaCode'].match?(/\s/)
```

Comment:
same as above - can we remove `&& unit_phone['areaCode'].match?(/\s/)` or check for presence?